src/components/output/OutputGestureStack.svelte

```diff
@@ -204,7 +208,7 @@
       <input
         class="h-25 rotate-90 accent-primary"
         type="range"
-        orient="vertical"
+        {...noTypeCheckNonStandardOrientProp('vertical')}
```

Comment:
[orient](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#orient) is a non-standard prop that seems to have been removed from the type definitions so I've added a workaround

Reply:
This has been a pain for a while. It's a shame we have to make this workaround, but the support for vertical range inputs are sorely lacking, and I suspect at some point we will make our own vertical slider to overcome the issue. It will do for now :)